build/run.js

```diff
@@ -517,7 +519,37 @@ async function installJsDeps() {
     }
 }
 
-async function runCommand(command,argv) {
+async function downloadJsAssets() {
+    const workdir = path.join(paths.root, '.assets-temp')
+    await cmd.with_cwd(paths.root, async () => {
+        await cmd.run('rm', ['-rf', workdir])
+        await cmd.run('mkdir', ['-p', workdir])
+    })
+    const ideAssetsMainZip = 'ide-assets-main.zip'
+    const unzippedAssets = path.join(workdir, 'ide-assets-main', 'content', 'assets')
+    const jsLibAssets = path.join(paths.js.lib.content, 'assets')
+
+    await cmd.with_cwd(workdir, async () => {
+        await cmd.run('curl', [
+            '--retry',
+            '4',
+            '--retry-connrefused',
+            '-fsSL',
+            '-o',
+            ideAssetsMainZip,
+            'https://github.com/enso-org/ide-assets/archive/refs/heads/main.zip',
+        ])
```

Comment:
is it working on Windows?

Reply:
Windows PowerShell has `cp`, `rm` and `mkdir` aliases that imitate bash commands. But I cannot test the script right now.